plugins/github/api.mli

```diff
@@ -31,6 +31,7 @@ module Commit : sig
   val compare : t -> t -> int
   val set_status : t Current.t -> string -> Status.t Current.t -> unit Current.t
   val uri : t -> Uri.t
+  val ref_title : t -> string
```

Comment:
```suggestion
  val pr_title : t -> string option
```
What about this? The title only makes sense for a Pull Request.

Reply:
We also want the name of the branch: we might need another name...